File: ungs-docsys-ms/src/main/java/com/ungs/docsys/dtos/EducationResponseDto.java
package com.ungs.docsys.dtos;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class EducationResponseDto {
    private String instituteName;
    private String degreeLevel;
    private String degree;
    private String fieldOfStudy;
}

File: ungs-docsys-bff/src/modules/resume-user/dtos/education-response.dto.ts
import { ApiProperty } from '@nestjs/swagger';

export class EducationResponseDto {
  @ApiProperty({ example: 'University of Example' })
  instituteName: string;

  @ApiProperty({ example: 'Bachelor' })
  degreeLevel: string;

  @ApiProperty({ example: 'Computer Science' })
  degree: string;

  @ApiProperty({ example: 'Software Engineering' })
  fieldOfStudy: string;
}
